package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.61.0",
   "description": "Official monday.com UI resources for application development in React.js",
   "main": "./dist/main.js",
+  "types": "./dist/types.d.ts",
```

Comment:
Cannot be added until all components have types.

Reply:
yes it was added by mistake will removed